crates/oxc_cli/src/lint/runner.rs

```diff
@@ -78,7 +78,7 @@ impl LintRunner {
                         Some(category) => rules.retain(|rule| rule.category() != category),
                         None => {
                             if name_or_category == "all" {
-                                rules.drain();
+                                rules.clear();
```

Comment:
warning: `drain` used to clear a `HashSet`
  --> crates/oxc_cli/src/lint/runner.rs:81:39
   |
81 | ...                   rules.drain();
   |                             ^^^^^^^ help: try: `clear()`
   |